.rubocop.yml

```diff
@@ -29,6 +29,9 @@ Style/FormatStringToken:
   Exclude:
     - spec/**/*
 
+Style/MutableConstant:
+  EnforcedStyle: strict
```

Comment:
This setting is unnecessary to match the default `EnforcedStyle: literals`.

Reply:
Thank you for the explanation. I think it is preferable to show that default `literals` works correctly on the RuboCop project.

Reply:
I think that the name `strict` mode is a good name and I think that setting it as an alternative is a good option. However, it seems to be too strict as default.

Reply:
Can you elaborate on why you think it is too strict for default? My end goal would be to hopefully have `strict` mode become the only option for this cop.

We are all aware of the issues with mutable constants, and I find that the current implementation of the cop is generally too lenient on freezing constants since it really only captures literals. There are very few reasons that someone would want a mutable constant. It is annoying that we have to add `freeze` all over the place, but the safety guarantee is worth it.

Reply:
I think immutability is a very useful concept in general. At the same time, I've never seen a Ruby code base that is stringent about the use of `#freeze`. I also never encountered a problem where having frozen an object would have saved me. This is, of course, based entirely in personal experience, but for that reason I would also be hesitant to make it the default. I'd consider it advanced usage.

Reply:
> I also never encountered a problem where having frozen an object would have saved me.

I unfortunately have, but it was a literal that would be picked up by the default setting on this cop. 

I get the argument for this being considered advanced usage. I will go ahead and disable `strict` mode on the RuboCop project. Should I also revert the change for freezing more constants in the project?

Reply:
> I've never seen a Ruby code base that is stringent about the use of #freeze. I also never encountered a problem where having frozen an object would have saved me.

I have never seen Rails, Bundler and other projects too (Of course I think that this depends on the scope being watched).

> Should I also revert the change for freezing more constants in the project?

Yes. Would you please revert the changed codes?